src/app/file-manager/models/item.ts

```diff
@@ -19,13 +18,22 @@ export class Item {
   transferred: number;
   remaining: number;
   bucketName: string;
+  workspaceName: string;
   prefix: string;
   delimiter: string;
   open: boolean;
+  selected: boolean;
+  indeterminate: boolean;
   namespace: string;
+  parent?: Item;
+  children?: Item[];
+  childCount?: number;
```

Comment:
This is set but it doesn't seem to be used.

Reply:
children is used, parent and childCount don't so will be removed.